src/components/MainPage.jsx

```diff
@@ -0,0 +1,240 @@
+import React, { useState, useEffect } from 'react'
+
+import { styled } from '@mui/material'
+
+import { MainContent } from './MainContent'
+
+import img from '../assets/images/main-like.png'
+
+import images from '../assets/images/main-dis.png'
+
+import image from '../assets/images/main-loy.png'
+
+const DivContainer = styled('div')`
```

Comment:
If the elements have similar styles, you can use only one container for this, for example:  const DivContainer = styled('div')`
   display: flex;
`